packages/pangea-django/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "pangea-django"
+version = "1.0.0"
+description = "Pangea Authentication Addon for Django"
+authors = ["Govind Salinas <govind.salinas@pangea.cloud>"]
+license = "MIT"
+readme = "README.md"
+homepage = ""
+repository = ""

```

Comment:
Fill in `homepage` and `repository`, otherwise this cannot be built with latest poetry (v1.7.1). Ref. #176.